src/Compatibility/Microsoft.DotNet.ApiCompat.Tool/Program.cs

```diff
@@ -0,0 +1,286 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using Microsoft.DotNet.ApiCompatibility.Logging;
+
+namespace Microsoft.DotNet.ApiCompat.Tool
+{
+    class Program
+    {
+        static int Main(string[] args)
+        {
+            // Global options
+            Option<string?> suppressionFileOption = new("--suppression-file",
```

Comment:
NIT: perhaps add a comment that mentions that this and the task-params should be kept in sync. Just for future reference in case someone is updating one but forgets to update the other